### PR TITLE
Issue-4059 changed color of Open Scenario in Hover mode

### LINKED ISSUE #4059: Text is not Visible when Hovering the Card in Dark Mode
### Description

go to this page https://layer5.io/learn/service-mesh-labs and turn on Dark Mode and hover over any card the text ( Open Scenario ) is not clearly visible in when we hover it in Dark Mode 

### Expected Behavior

That text should give hover effect as it gives in Light Mode that text should change its color to pale green type as it in website.

### Screenshots

https://user-images.githubusercontent.com/86219935/230066488-848a21df-3fcc-40d4-b8e5-b295c597c866.mp4

### Environment:
- Host OS: Windows 10
- Browser: Chrome

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by hirentimbadiya**
**Description**

This PR fixes #4059 

**Notes for Reviewers**

https://user-images.githubusercontent.com/86219935/230394345-75be4055-45e5-4f2c-8f05-ccc33c9e4743.mp4

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by hirentimbadiya** [comment]
Please Review this PR , @leecalcote , @Chadha93 , @Nikhil-Ladha  , @Shivam-AfA

**Comment by leecalcote** [comment]
@hirentimbadiya review the lighthouse checks for this change and confirm that no issues are reported (that is, assuming that lighthouse checks are covering this page, @ayushthe1 )

**Comment by l5io** [comment]
🚀 Preview for commit 7c03ba79c010f8f9084c10b707dbf62036ceedd0 at: https://642ecf0f4dc2124586f1b931--layer5.netlify.app

**Review by ayushthe1 — CHANGES_REQUESTED** [review]
Just undo the changes you have done in package-lock.json. Else it's good.

File: package-lock.json [hunk 1]
```diff
@@ -38,9 +38,9 @@
         "gatsby-plugin-mdx": "^3.20.0",
         "gatsby-plugin-meta-redirect": "^1.1.1",
         "gatsby-plugin-preload-fonts": "^3.24.0",
-        "gatsby-plugin-robots-txt": "^1.8.0",
+        "gatsby-plugin-robots-txt": "^1.7.1",
         "gatsby-plugin-sharp": "^4.19.0",
-        "gatsby-plugin-sitemap": "^6.8.0",
+        "gatsby-plugin-sitemap": "^5.19.0",
```
[the remark this answers is not in the document]
Comment (hirentimbadiya) [reply package-lock.json:43]: when i ran make site command it automatically changed to this. should i revert it back ? @ayushthe1

**Comment by hirentimbadiya** [comment]
> Just undo the changes you have done in package-lock.json. Else it's good.

Okay!

**Comment by l5io** [comment]
🚀 Preview for commit a44d449f4c23eec2cafe53336a1728ab85902613 at: https://642eea0c346c8600bf0004d5--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 3ff31325e56a7f9a93326c6ca53bfee9fef44051 at: https://642fa3a45b894a626a8be1b4--layer5.netlify.app

**Review by ayushthe1 — APPROVED** [review]
good to go now :+1:

**Comment by hirentimbadiya** [comment]
@leecalcote can you review this now ?

**Review by thisiskaransgit — APPROVED** [review]
Looks good to merge, thank you for your contribution :tada:

**Review by leecalcote — APPROVED** [review]
Thank you, gents.

**Comment by l5io** [comment]
🚀 Preview for commit 00454c76e471678c1a406cb1a7d1eebe3dd71148 at: https://6430e505f7271a46b0cfd099--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 05b7ef6c9468edaddda1cdb468215372b117defd at: https://643243bea308c14498b303a6--layer5.netlify.app

**Pull request merged**